vendor/github.com/go-xorm/xorm/statement.go

```diff
@@ -160,7 +161,7 @@ func (statement *Statement) And(query interface{}, args ...interface{}) *Stateme
 			}
 		}
 	default:
-		// TODO: not support condition type
+		statement.lastError = ErrConditionType
```

Comment:
It appears this is the cause of the trouble in unit tests. I think if you change:

https://github.com/go-gitea/gitea/blob/02a91a9393ae1452dbcc62fc8f316514cbdcc6b5/models/consistency.go#L42

to:

```go
assert.NoError(t, x.Table(bean).Find(ptrToSliceValue.Interface()))
```

that might fix it. But I'm not completely sure I understand what consistency is doing.

Reply:
done.